spec/helpers/govuk_link_helper_spec.rb

```diff
@@ -80,11 +80,40 @@ def url_for(path)
       end
     end
 
-    context "when new_tab: true" do
-      let(:kwargs) { { new_tab: true } }
+    describe "new_tab" do
+      let(:expected_new_tab_attributes) { { href: "/world", class: "govuk-link", target: "_blank", rel: "noreferrer noopener" } }
 
-      specify "the new tab attributes are present on the link" do
-        expect(subject).to have_tag("a", text: "hello", with: { href: "/world", class: "govuk-link", target: "_blank", rel: "noreferrer noopener" })
+      context "when new_tab: true" do
+        let(:kwargs) { { new_tab: true } }
+
+        specify "the link has the new tab attributes and the default 'new tab' text is appended" do
+          expect(subject).to have_tag("a", text: "hello (opens in new tab)", with: expected_new_tab_attributes)
```

Comment:
Up to you, but I always think these tests are easier to follow if the expectations are explicitly defined inline, at the expense of duplication?  Could also only test the attributes that are different from the other link tests which are tested further up?

```suggestion
          expect(subject).to have_tag("a", text: "hello (opens in new tab)", with: { target: "_blank", rel: "noreferrer noopener" })
```

Reply:
I'll leave it for now just because the attributes are a bit wordy and the variable is defined within the parent group.